### PR TITLE
Create banzhuxiaoshuo.xml

### PR DESCRIPTION
**Pull request by 15556133244**

### CODE REVIEW

File: src/chrome/content/rules/banzhuxiaoshuo.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="banzhuxiaoshuo">
+	<target host="banzhuxiaoshuo.com" />
+	<target host="www.banzhuxiaoshuo.com" />
+	<target host="m.banzhuxiaoshuo.com" />
+
+	<rule from="^http:"
+			to="https:" />
```
Comment (pipboy96) [inline src/chrome/content/rules/banzhuxiaoshuo.xml]: Single line (no `\n`):
`<rule from="^http:" to="https:" />`

File: src/chrome/content/rules/banzhuxiaoshuo.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="banzhuxiaoshuo">
+	<target host="banzhuxiaoshuo.com" />
+	<target host="www.banzhuxiaoshuo.com" />
+	<target host="m.banzhuxiaoshuo.com" />
+
```
Comment (pipboy96) [inline src/chrome/content/rules/banzhuxiaoshuo.xml]: `<securecookie host=".+" name=".+" />`

File: src/chrome/content/rules/banzhuxiaoshuo.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+	<securecookie host=".+" name=".+" />
+	<rule from="^http:" to="https:" />
```
Comment (pipboy96) [inline src/chrome/content/rules/banzhuxiaoshuo.xml]: Add `\n` between `<securecookie>` and `<rule>` tags.

**Comment by 15556133244** [comment]
@pipboy96

**Comment by pipboy96** [comment]
?

**Comment by ivysrono** [comment]
you  approved these changes 3 months ago

**Comment by pipboy96** [comment]
@15556133244 Did you close this PR by accident?

**Pull request closed without merging**